src/domains/gps-app/00_alert_gpd_core.tf

```diff
@@ -179,3 +179,37 @@ AzureDiagnostics
     threshold = 1
   }
 }
+
+## GPD-SEND communication status ##
+resource "azurerm_monitor_scheduled_query_rules_alert" "opex_pagopa-gpd-send-communication-exception" {
+  count               = var.env_short == "p" ? 1 : 0
+  resource_group_name = "dashboards"
+  name                = "pagopa-${var.env_short}-opex_pagopa-gpd-send-comm-exception"
+  location            = var.location
+
+  action {
+    action_group           = [data.azurerm_monitor_action_group.slack_pagopa_pagamenti_alert.id]
+    email_subject          = "GPD-SEND-STATUS"
+    custom_webhook_payload = "{}"
+  }
+
+  data_source_id = data.azurerm_application_insights.application_insights.id
+  description    = "An exception occurred when GPD tries to call SEND for updating notifications, it does NOT cause failures to asynchronous payments."
+  enabled        = true
+  query = (<<-QUERY
+let threshold = 0.99;
+traces
```

Comment:
Why traces instead of exceptions?

Reply:
Because the log added [here](https://github.com/pagopa/pagopa-debt-position/pull/301) are saved as traces. We catch the exception when communication with SEND fail.